internal/connect/command_offset_describe.go

```diff
@@ -0,0 +1,138 @@
+package connect
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+	"github.com/tidwall/pretty"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type OffsetConnectorOut struct {
+	Id       string `human:"ID"`
+	Name     string `human:"Name"`
+	Offsets  string `human:"Offsets"`
+	Metadata string `human:"Metadata"`
```

Comment:
I assume we want to match serialized output here?
```suggestion
	Offsets  string `human:"Offsets,omitempty"`
	Metadata string `human:"Metadata,omitempty"`
```

Reply:
Just curious though, if both offsets and metadata are empty, don't you think the user would be confused if they only saw "ID" and "Name"?

Reply:
removed `,omitempty` from `serializedOffsetConnectorOut`